src/components/Header/MobileNav.tsx

```diff
@@ -50,6 +52,15 @@ export const MobileNav: FunctionComponent<MobileNavProps> = ({
                   },
                 ]}
               />
+
+              <NavLink
+                color="blue.800"
+                fontSize="1rem"
+                fontWeight="bold"
+                to={ROUTES.CONTRIBUTE}
+              >
+                Contribute
+              </NavLink>
```

Comment:
super small nitpick, but can we make the tap area for this link span the full horizontal width for mobile users?

Reply:
Yeah this link will look / feel more like the buttons it is placed next to

<img width="324" alt="Screen Shot 2021-11-18 at 8 53 38 AM" src="https://user-images.githubusercontent.com/8749859/142460473-71b01115-5562-4a46-a5f1-3ed2e872156b.png">